packages/windows/src/RPA/Windows/keywords/locators.py

```diff
@@ -277,6 +288,8 @@ def get_element(
             Set Value   ${element}  note to myself
         """
         # TODO. Add examples
+        if isinstance(locator, str):
+            locator = self._load_by_alias(locator)
```

Comment:
This is a good place, well done! (since this is the source of truth for getting elements)